alleyinteractive/wp-newsletter-builder
# Issue-182: Trim down @wordpress dependencies to those only used in code

## Summary

Fixes #182
Fixes #183 

This Pull Request addresses the issue titled "Trim down @wordpress dependencies to those only used in code" by removing and upgrading the NPM `@wordpress` related dependencies as needed. This ensures that only the dependencies that are actively used in the codebase are included, streamlining the development process and minimizing unnecessary bloat.

## Use Case

Ensures that when a developer runs `npm install`, only the `@wordpress` dependencies that are used in code are installed. This change not only removes unused dependencies but also upgrades existing ones to their latest versions, enhancing the application's security and performance.

## Changes

- Removed unused `@wordpress` dependencies from `package.json`.
- Upgraded versions of the remaining `@wordpress` dependencies to their latest.

## Test Instructions

1. Checkout this branch.
2. Run `npm install`.
3. Verify that only the necessary `@wordpress` dependencies are installed and that the application builds correctly. Also, ensure that the upgraded dependencies do not introduce any breaking changes.

## Additional Information

For more details, see the original issue here: https://github.com/alleyinteractive/wp-newsletter-builder/issues/182
